dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201013AddNewColumnsToIdentifierTable.java

```diff
@@ -0,0 +1,68 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import com.dotmarketing.util.Logger;
+import java.sql.SQLException;
+import java.util.List;
+
+public class Task201013AddNewColumnsToIdentifierTable extends AbstractJDBCStartupTask {
+    @Override
+    public boolean forceRun() {
+        try {
+            final DotDatabaseMetaData dbMetadata = new DotDatabaseMetaData();
+            return !dbMetadata.hasColumn("identifier", "owner") &&
+                    !dbMetadata.hasColumn("identifier", "create_date") &&
+                    !dbMetadata.hasColumn("identifier", "asset_subtype");
+        } catch (SQLException e) {
+            Logger.error(this, e.getMessage(),e);
+            return false;
+        }
+    }
+
+
+    @Override
+    public String getPostgresScript() {
+        return "alter table identifier add owner varchar(255);" +
+                "alter table identifier add create_date timestamp;" +
+                "alter table identifier add asset_subtype varchar(255);" +
+                "create index idx_identifier_asset_subtype on identifier (asset_subtype);";
```

Comment:
Codacy found an issue: [The String literal "create index idx_identifier_asset_subtype on identifier (asset_subtype);" appears 4 times in this file; the first occurrence is on line 29](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6327767)